backend/src/api/organization/dto/create-access-code.dto.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  IsDate,
+  IsOptional,
+  IsString,
+  MaxLength,
+  MinLength,
+} from 'class-validator';
+
+export class CreateAccessCodeDto {
+  @IsString()
+  @MaxLength(10)
```

Comment:
If these are the real validations we need to change the story: https://wearetribus.atlassian.net/browse/TEO-36

Reply:
Updated the story.